composables/getters/facetGetters.ts

```diff
@@ -0,0 +1,34 @@
+import { buildBreadcrumbs } from "../helpers/buildBreadcrumbs"
+import { Breadcrumb } from "~~/pages/types"
+import { Maybe, PrCategory } from "~~/server/types/GraphQL"
```

Comment:
use @/ instead of ~~/ everywhere for code consistency purpose

Reply:
Resolved